custom_connectors/oauth2/ephesoft_connector.rb

```diff
@@ -0,0 +1,225 @@
+{
+  title: 'Ephesoft Semantik for Invoices',
+
+  connection: {
+    fields: [
+      {
+        name: 'client_id',
+        optional: false
+      },
+      {
+        name: 'client_secret',
+        optional: false,
+        control_type: 'password'
+      }
+    ],
+    authorization: {
+      type: "oauth2",
+      authorization_url: lambda do |connection|
+        params = {
+          client_id: connection["client_id"],
+          response_type: "code",
+          redirect_uri: "https://www.workato.com/oauth/callback",
+          scope: "admin"
+        }.to_param
+
+        "https://api.us.ephesoft.io/v1/auth/oauth2/authorize?" + params
+      end,
+
+      acquire: lambda do |connection, auth_code|
+       response = post("https://api.us.ephesoft.io/v1/auth/oauth2/token").
+                    payload(
+                      code: auth_code,
+                      grant_type: "authorization_code",
+                      client_id: connection["client_id"],
+                      client_secret: connection["client_secret"],
+                      redirect_uri: "https://www.workato.com/oauth/callback"
+                    ).
+                    request_format_www_form_urlencoded
+        [
+          {
+            access_token: response["access_token"],
+            refresh_token: response["refresh_token"]
+          }
+        ]
+      end,
+
+      refresh_on: [401, 403],
+
+      refresh: lambda do |connection, refresh_token|
+        response = post("https://api.us.ephesoft.io/v1/auth/oauth2/refresh").
+                      payload(
+                        grant_type: "refresh_token",
+                        refresh_token: refresh_token,
+                        client_id: connection["client_id"],
+                        client_secret: connection["client_secret"],
+                        redirect_uri: "https://www.workato.com/oauth/callback"
+                      ).
+                      request_format_www_form_urlencoded
+        [
+          { # Tokens hash
+            access_token: response["access_token"],
+            refresh_token: response["refresh_token"]
+          },
+          { instance_id: nil } # Optional. Will be merged into connection hash
+        ]
+      end,
+
+      apply: lambda do |connection, access_token|
+        headers("Authorization": "Bearer #{access_token}")
+      end
+    }
+  },
+  test:
+    lambda do |connection|
+      get("https://api.us.ephesoft.io/v1/settings/integrations/configurations")
+    end,
+  actions: {
+    # Future action code here
+  },
+  triggers: {
+    new_message: {
+      title: 'When a Semantik invoice has completed review',
+      description: 'Triggers a flow using data from a completed Semantik invoice.',
```

Comment:
Normally we follow this terminology:
`Triggers when a Semantik Invoice is completed`

Reply:
Updated title to follow standard in latest commit.